src/routes/routeConstants.ts

```diff
@@ -21,3 +22,4 @@ export const ACTIONS_PAGE_ROUTE = `${COLONY_HOME_ROUTE}/tx/:transactionHash`;
 export const COIN_MACHINE_ROUTE = `${COLONY_HOME_ROUTE}/buy-tokens/:transactionHash?`;
 export const UNWRAP_TOKEN_ROUTE = `${COLONY_HOME_ROUTE}/unwrap-tokens`;
 export const CLAIM_TOKEN_ROUTE = `${COLONY_HOME_ROUTE}/claim-tokens`;
+export const EXPENDITURE_ROUTE = `${COLONY_HOME_ROUTE}/create-expenditure`;
```

Comment:
@agnieszkajarosikloj Can we please go with `/expenditure/create`, then when an expenditure has a unique URL it goes to `/expenditure/{UID}`